lib/sqlalchemy/ext/linter.py

```diff
@@ -0,0 +1,114 @@
+import collections
```

Comment:
**mike bayer** (zzzeek) wrote:

want to try for any docs?  I can do them at some point if not